### ISSUE TITLE
Watch bounty applications

### ISSUE THREAD

**Issue by lafkpages** [lafkpages/XL-Replit#13]
**Is your feature request related to a problem? Please describe.**
I'd like to get notifications when a specific bounty gets an application, or the bounty is closed.

**Describe the solution you'd like**
A button or option to watch a bounty.

**Describe alternatives you've considered**
Other people's bounty viewer Repls, but I'd like this to be part of the extension.

**Additional context**

**Comment by DillonB07** [comment]
Sounds interesting. Maybe do it similar to GH with the ability to change different types of watching. Also, please do this for repls as well with the ability to be notified on all or only comments, updates and mentions.

**Comment by lafkpages** [comment]
You know how Replit Sitemods can send custom notifs? Can users send them to themselves? Or would the extension have to just inject fake notifs

**Comment by lafkpages** [comment]
Yeah, they can't. But I meant only to themselves, so spam wouldn't be a problem.
